src/types/onyx/TransactionViolation.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @module TransactionViolation
+ * @description Transaction Violation
+ */
+
+/**
+ * Names of the various Transaction Violation types
+ */
+type ViolationName =
+    | 'perDayLimit'
+    | 'maxAge'
+    | 'overLimit'
+    | 'overLimitAttendee'
+    | 'overCategoryLimit'
+    | 'receiptRequired'
+    | 'missingCategory'
+    | 'categoryOutOfPolicy'
+    | 'missingTag'
+    | 'tagOutOfPolicy'
+    | 'missingComment'
+    | 'taxRequired'
+    | 'taxOutOfPolicy'
+    | 'billableExpense';
+
+type ViolationType = string;
```

Comment:
Why do we need this new type? Can we keep it simple and use string?
```
type TransactionViolation = {
    type: string;
```

Reply:
That's a stub--I'll remove it.